extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -49,6 +49,7 @@ public static function replaceInfoboxMarkers( &$parser, &$text ) {
 	 * @return string
 	 * @throws UnimplementedNodeException when node used in markup does not exists
 	 * @throws XmlMarkupParseErrorException xml not well formatted
+	 * @throws notValidInfoboxAttributesErrorException when unsupported attributes exist in params array
```

Comment:
For convenience I'd rename it to InvalidInfoboxAttributeException as this name is both very long, and uses both Error and Exception in the name (like the above one)